app/utils/commands.js

```diff
@@ -0,0 +1,241 @@
+const { UntilMorning } = require('./untilMorning')
+
+const allOptions = {
+  title: {
+    long: '--title',
+    short: '-T',
+    description: 'Specify title for next break (Long or Mini)',
+    withValue: true
+  },
+  text: {
+    long: '--text',
+    short: '-t',
+    description: 'Specify text for next break (Long Break only)',
+    withValue: true
+  },
+  skip: {
+    long: '--skip',
+    short: '-s',
+    description: 'Skip directly to this break (Long or Mini)',
+    withValue: false
+  },
+  duration: {
+    long: '--duration',
+    short: '-d',
+    description: 'Specify duration for pausing breaks (Pause only) [infinitely|until-morning|seconds (as number)]',
+    withValue: true
+  }
+}
+
+const allCommands = {
+  help: {
+    description: 'Show this help message'
+  },
+  version: {
+    description: 'Show current stretchly version'
+  },
+  reset: {
+    description: 'Reset breaks'
+  },
+  pause: {
+    description: 'Pause breaks',
+    options: [allOptions.duration]
+  },
+  resume: {
+    description: 'Resume from a pause'
+  },
+  toggle: {
+    description: 'Toggle breaks between resume/paused'
+  },
+  mini: {
+    description: 'Skips to next Mini Break',
+    options: [allOptions.title, allOptions.skip]
+  },
+  long: {
+    description: 'Skips to next Long Break',
+    options: [allOptions.text, allOptions.title, allOptions.skip]
+  }
+}
+
+const allExamples = [{
+  cmd: 'stretchly pause',
+  description: 'Pause breaks indefinitly'
+},
+{
+  cmd: 'stretchly pause -d 3600',
+  description: 'Pause breaks for one hour'
+},
+{
+  cmd: 'stretchly mini -T "Stretch up !" --skip',
+  description: 'Skips to next Mini Break with "Stretch up!" title'
+},
+{
+  cmd: 'stretchly long -T "Stretch up !" -t "Go stretch !"',
+  description: 'Set next long break title to "Stretch up !" and text to "Go stretch !"'
+}]
+
+// Parse cmd line, check if valid and put variables in a dedicated object
+class Command {
+  constructor (input, version) {
+    this.version = version
+    this.supported = allCommands
+
+    this.parse(input)
+  }
+
+  parse (input) {
+    this.command = input[0]
+
+    if (!this.supported[this.command]) {
+      throw new Error(`Error: command ${this.command} is not supported`)
```

Comment:
I usually run with `npm run dev`. Could we make it run fine with this? 

```
"dev": "cross-env NODE_ENV=development electron ./app --enable-logging --remote-debugging-port=9222",
```

It's throwing `Error: Error: command --enable-logging is not supported` now.

Reply:
It's thrown if you have another primary stretchly instance running. The code pass in https://github.com/papey/stretchly/blob/features/cmdline-automation/app/main.js#L45 this path. If you need multiple stretchly instance, this `gotTheLock` might not be a good option.

Reply:
What I mean is: 

This runs fine:`npm start -- pause`
This does not run fine `npm run dev -- pause`

I guess we need to add `--enable-logging --remote-debugging-port=9222` to valid comands, but ignore them.